hack/check-gh-chart-versions.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+TEXT_RED='\033[0;31m'
+TEXT_RESET='\033[0m'
+TEXT_GREEN='\033[0;32m'
+
+target_version=$1
+if [[ $# -eq 0 ]]; then
+    echo "Release version argument is required"
+    echo
+    echo "Usage:  ${0} <release version>"
+    exit 1
+fi
+
+chart_dir=$(pwd)/charts
```

Comment:
Always double quote variables, including subshells. No naked $ signs. More on that here: http://mywiki.wooledge.org/Quotes